qupath-core-processing/src/main/java/qupath/lib/scripting/QP.java

```diff
@@ -5187,5 +5189,90 @@ private static void warnAboutSetter(String name) throws UnsupportedOperationExce
 				 + "or default a local variable with 'def {}'", name, name, name);
 		 throw new UnsupportedOperationException(name + " cannot be set!");
 	 }
-	
+
+	/**
+	 * Simplify all annotations to a given threshold
+	 * <p>
+	 *     See {@link ShapeSimplifier#simplifyPolygon(PolygonROI, double)} )} for details.
+	 * </p>
+	 * @param altitudeThreshold altitude value for simplification
+	 */
+	public static void simplifyAllAnnotations(double altitudeThreshold) {
+		 simplifySpecifiedAnnotations(getAnnotationObjects(), altitudeThreshold);
+	}
+
+	/**
+	 * Simplify the currently selected annotations to a given threshold
+	 * <p>
+	 *     See {@link ShapeSimplifier#simplifyPolygon(PolygonROI, double)} )} for details.
```

Comment:
FYI there's a standard annotation for that - at the end of the javadoc, you can add
```
@see ShapeSimplifier#simplifyPolygon(PolygonROI, double)
```

Reply:
Ah! thanks